src/main/java/tools/image/CenterPicker.java

```diff
@@ -327,7 +326,7 @@ private static String getValue(final String arg) {
   private static void handleCommandLineArgs(final String[] args) {
     // arg can only be the map folder location.
     if (args.length == 1) {
-      String value;
+      final String value;
       if (args[0].startsWith(TRIPLEA_MAP_FOLDER)) {
         value = getValue(args[0]);
       } else {
```

Comment:
Was tempted to loop this in with #1919, I personally would like to see where we go with the beta map making tool kit before updating it further.